### PR TITLE
Fix publish build and dependencies

### PR DESCRIPTION
**Pull request by vitaliystoliarovcc**
### Description:

Asana ticket: https://app.asana.com/0/1125859137800433/1200174906704565/f
+ other minor changes are described below.

### CODE REVIEW

**Review by avsek477 — CHANGES_REQUESTED** [review]
we should move the babel packages from dependencies to devDependencies as well ?

**Comment by robertkowalski** [comment]
when you test a publish, is the `dist` folder included in the package tarball?

**Comment by vitaliystoliarovcc** [comment]
> when you test a publish, is the dist folder included in the package tarball? 

No, the same issue is present on https://www.jsdelivr.com/package/npm/bfx-hf-indicators?version=2.0.5
Seems like 'postinstall' fixes it on install stage

Regarding this PR, a lot will depend on how you execute `npm publish` command.

**Comment by vitaliystoliarovcc** [comment]
In fact, the prepublishOnly` script is not the only change required.

At the moment I see the following problems:
1. `build` script copies package.json, but it is not used for publication (because of step 2)
2. The root folder with the source code is published in npmjs (since package.json is copied to dist, we can only publish the content from this folder, which is common practice since we don't need test directories or any configuration files for distribution)

What do you think?

**Comment by robertkowalski** [comment]
- i like both solutions and both work well
 - for #2 i like that the source is still available in the local npm modules, which can make local debugging a bit easier

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ package-lock.json
 tags
 .nyc_output/
 coverage/
+*.tgz
```
Comment (robertkowalski) [inline .gitignore:16]: missing newline at end of file

you can enable it as an editor setting

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.github
+examples
+test
+*.tgz
+.travis.yml
+.babelrc
```
Comment (robertkowalski) [inline .npmignore:6]: same here

**Review by robertkowalski — CHANGES_REQUESTED** [review]
looks nice, two small nits with EOF newlines

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**